pages/_app.tsx

```diff
@@ -1,8 +1,50 @@
 import { AppProps } from "next/app";
-import "../styles/globals.css";
+import Head from "next/head";
+import { AppShell, Header, MantineProvider, Navbar } from "@mantine/core";
+import { NotificationsProvider } from "@mantine/notifications";
+import { ResourceCounts } from "../components/ResourceCounts";
 
-function MyApp({ Component, pageProps }: AppProps) {
-  return <Component {...pageProps} />;
-}
+export default function App(props: AppProps) {
+  const { Component, pageProps } = props;
 
-export default MyApp;
+  return (
+    <>
+      <Head>
+        <title>Test server frontend</title>
+        <meta name="viewport" content="minimum-scale=1, initial-scale=1, width=device-width" />
+      </Head>
+      <MantineProvider
+        withGlobalStyles
+        withNormalizeCSS
+        theme={{
+          /** Put your mantine theme override here */
+          colorScheme: "light",
+        }}
+      >
+        <NotificationsProvider position="top-center">
+          <AppShell
+            padding="md"
+            navbar={
+              <Navbar width={{ base: 300 }} height={800} p="xs">
```

Comment:
```suggestion
              <Navbar width={{ base: 300 }} p="xs">
```

Part 1 of a 2 part comment, but I think we can get rid of the fixed height on the navbar, and just allow it to take up the full screen. Part 2 will be on different code to allow it to scroll when overflowed